lib/ansible/modules/files/lineinfile.py

```diff
@@ -470,7 +471,7 @@ def main():
             create=dict(type='bool', default=False),
             backup=dict(type='bool', default=False),
             firstmatch=dict(default=False, type='bool'),
-            validate=dict(type='str'),
+            validate=dict(type='str')
```

Comment:
Keep the trailing comma. Having it there makes for a cleaner diff if another line is added below.